recipes/python-blosc/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.3.2" %}
+
+package:
+  name: python-blosc
+  version: {{ version }}
+
+source:
+  url: https://github.com/Blosc/python-blosc/archive/v{{ version }}.tar.gz
+  sha256: 63d1cd1da14087fa69bffc513b25d59deb24f45c65e4406acafaaa6ec9bd7cf2
+
+build:
+  number: 0
+  skip: True  # [win]
+
```

Comment:
Since you are not building on Windows you can remove the `build.sh` file and add:

`script: python setup.py install --blosc=$PREFIX --single-version-externally-managed --record record.txt`

here.